model/mllama/model.go

```diff
@@ -0,0 +1,90 @@
+package mllama
+
+import (
+	"github.com/ollama/ollama/ml"
+	"github.com/ollama/ollama/ml/nn"
+	"github.com/ollama/ollama/model"
+)
+
+type Model struct {
+	model.Base
+
+	*VisionModel `ggml:"v,vision"`
```

Comment:
```suggestion
	*VisionModel `gguf:"v,vision"`
```

`ggml` may seem like it's tied to that backend - would we want to make this either `tensor` or `gguf`?